<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>330</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Вычислительная математика, Беленко А.В.</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QLabel" name="TypeOfTask">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>10</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип решаемой задачи:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TaskButton1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Решение ДУ методами
Эйлера и Рунге-Кутты</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="TaskButton2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Решение ДУ
 второго порядка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TaskButton3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>200</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Решение системы ДУ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TaskButton5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>129</y>
     <width>200</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>За какое время
сила тока в цепи
достигнет максимума?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TaskButton6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>pass</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TaskButton4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>200</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>За сколько минут
 остынет кофе?</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>TaskButton1</sender>
   <signal>clicked()</signal>
   <receiver>TaskButton1</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
